kratos-admin-ui/src/service/schema-service.test.ts

```diff
@@ -320,4 +323,64 @@ describe("test getSchemaFields", () => {
         expect(fields[1].type).toBe("string");
         expect(fields[2].type).toBe("string");
     })
+
+    test("test public metadata schema", () => {
+        const schema = {
+            "$id": "https://schemas.ory.sh/presets/kratos/identity.email.schema.json",
+            "title": "Person",
+            "type": "object",
+            "properties": {
+                "traits": {
+                    "type": "object",
+                    "properties": {
+                        "email": {
+                            "type": "string",
+                            "format": "email",
+                            "title": "E-Mail",
+                            "minLength": 3,
+                            "ory.sh/kratos": {
+                                "credentials": {
+                                    "password": {
+                                        "identifier": true
+                                    }
+                                },
+                                "verification": {
+                                    "via": "email"
+                                },
+                                "recovery": {
+                                    "via": "email"
+                                }
+                            }
+                        },
+                    },
+                    "required": ["emails"],
```

Comment:
thats a small thing, but while checking your code i just copied this schema in my kratos instance, but i failed to create a new identity, since the property is `email` not `emails` :)